src/js/stickyKeysAdjuster.js

```diff
@@ -0,0 +1,114 @@
+            // stickyKeysEnabled: boolean
+        },
+        modelRelay: {
+            source: "tryAccomodation",
+            target: "stickyKeysEnabled",
+            backward: "never",
+            singleTransform: {
+                type: "fluid.transforms.identity"
+            }
+        },
+        modelListeners: {
+            tryAccomodation: {
+                listener: "gpii.firstDiscovery.keyboard.stickyKeysAdjuster.tryAccomodationToggle",
+                args: ["{that}", "{change}.value"]
+            },
+            stickyKeysEnabled: {
+                listener: "gpii.firstDiscovery.keyboard.stickyKeysAdjuster.displayState",
+                args: ["{that}"]
+            }
+        },
+        listeners: {
+            "onCreate.setText": {
+                listener: "gpii.firstDiscovery.keyboard.stickyKeysAdjuster.renderText",
+                args: ["{that}"]
+            },
+            "onCreate.bindTry": {
+                "this": "{that}.dom.tryButton",
+                "method": "click",
+                "args": ["{that}.toggleTry"]
+            },
+            "onCreate.bindToggle": {
+                "this": "{that}.dom.accomodationToggle",
+                "method": "click",
+                "args": ["{that}.toggleStickyKeys"]
+            }
+        },
+        invokers: {
+            toggleStickyKeys: {
+                funcName: "gpii.firstDiscovery.keyboard.stickyKeysAdjuster.toggleState",
+                args: ["{that}", "stickyKeysEnabled"]
+            },
+            toggleTry: {
+                funcName: "gpii.firstDiscovery.keyboard.stickyKeysAdjuster.toggleState",
+                args: ["{that}", "tryAccomodation"]
+            }
+        }
+    });
+
+    gpii.firstDiscovery.keyboard.stickyKeysAdjuster.renderText = function (that) {
+        var resolveFn = that.msgResolver.resolve;
+
+        that.locate("accomodationInstr").text(resolveFn("stickyKeysAccomInstr"));
+        that.locate("description").html(resolveFn("stickyKeysInstructions"));
+        that.locate("tryButton").text(resolveFn("try"));
+        that.locate("accomodationName").text(resolveFn("stickyKeys"));
+        gpii.firstDiscovery.keyboard.stickyKeysAdjuster.displayState(that);
+    };
+
+    gpii.firstDiscovery.keyboard.stickyKeysAdjuster.tryAccomodationToggle = function (that, state) {
+        that.locate("tryButton").toggle(!state);
+        that.locate("accomodation").toggle(state);
+    };
+
+    gpii.firstDiscovery.keyboard.stickyKeysAdjuster.displayState = function (that) {
+        var state = that.model.stickyKeysEnabled;
+        var stateText = that.msgResolver.resolve(state ? "on" : "off");
+        var buttonText = that.msgResolver.resolve(state ? "turnOff" : "turnOn");
+        var tooltipText = that.msgResolver.resolve(state ? "turnOffTooltip" : "turnOnTooltip");
+        that.locate("accomodationState").text(stateText);
+        that.locate("accomodationToggle").text(buttonText);
+        if (that.tooltip) {
```

Comment:
I think the actual issue here is the timing of creation, where the model relay was fired before the tooltip subcomponent was created. I've split the function in two still, separating based on display vs tooltip changes, and passed in the tooltip via IoC to the appropriate function.